src/mem/PoolChunked.h

```diff
@@ -31,7 +31,7 @@ class MemPoolChunked : public MemImplementingAllocator
      \param stats   Object to be filled with statistical data about pool.
      \retval        Number of objects in use, ie. allocated.
      */
-    virtual int getStats(MemPoolStats * stats, int accumulate);
```

Comment:
Here and in similar places, please drop the misplaced/duplicated/incomplete method description or replace it with a `/* Allocator API */` comment.

Reply:
To keep this refactor small I have deferred these cleanups to PRs updating the other MemPoolX classes.